modulus/models/graphcast/graph_cast_net.py

```diff
@@ -183,13 +183,8 @@ def __init__(
         activation_fn = get_activation(activation_fn)
 
         # construct the graph
-        try:
-            self.graph = Graph(meshgraph_path, self.lat_lon_grid)
-        except FileNotFoundError:
-            raise FileNotFoundError(
-                "The icospheres_path is corrupted. "
-                "Tried using pymesh to generate the graph but could not find pymesh"
-            )
+        self.graph = Graph(self.lat_lon_grid, multimesh_level)
```

Comment:
I also wanted to call out, in L176, the definition of `longitudes` is not consistent with the ERA5 dataset, which I believe is typically `linspace(0, 360, 1440, endpoint = False)`. I don't think this actually makes a different in the behavior of the model, however.

Reply:
You are right, but I wanted to remain consistent with the paper. See equation A.4 on page 25: https://arxiv.org/pdf/2212.12794v1